data/v.json

```diff
@@ -9612,13 +9644,17 @@
                 "def": "the process of becoming a vapor",
                 "speech_part": "noun",
                 "synonyms": [
-                    " vaporization",
-                    "vaporisation",
-                    "evaporation",
-                    "vapour",
-                    "vapor"
+                    "vaporisation"
                 ]
             }
+        ],
+        "editors": [
+            "lefurjah"
+        ],
+        "contributors": [
+            "luxfactaest",
+            "msingle",
+            "zellerpress"
         ]
     },
     "vaporisation": {
```

Comment:
Is this even a word?

Reply:
vaporization/vaporisation? I believe so...

Reply:
British version!